road/validation_road.py

```diff
@@ -0,0 +1,113 @@
+import os

```

Comment:
less important, but suggest conforming to PEP8 import sorting guidelines: https://peps.python.org/pep-0008/#imports

isort or ruff formatters would take care of this